### DISCUSSION TITLE
Migration from Storybook

### DISCUSSION THREAD

**Discussion by sadn1ck** [creativecommons/vocabulary-components#17] [Ideas]
Quoting from #16

>The Storybook dependency, while exciting, has led to complexity and dependency bloat in the Vocabulary Components. We are working to restore this project to good health, including updating all dependencies. We are considering removing Storybook in order to make dependency management easier. We can consider how to use Storybook via a separate repository, but want to avoid intermixing it into this project.

Storybook, while an amazing tool for development, has been getting in the way more often than not. As such, we are looking at possible lightweight alternatives for Storybook.

Ideally we should be going for something with a large community support and active development to prevent any long term instability.

## Possible alternatives

- vuepress (based on vue 2, might need an upgrade when we upgrade our components to vue 3)
- vitepress (based on vue 3 and vite, project still in 0.x status)

Any and all suggestions welcome!

cc: @brylie @MuluhGodson

**Comment by MuluhGodson** [comment]
@sadn1ck Vuepress is a good option.

**Comment by brylie** [comment]
If we deprecate Storybook initially, it might make a Vue 3 migration easier. Then we can check out VitePress.

In the meantime, we should be able to move forward with VuePress, since it and VitePress should follow the same content structure. We can also preserve the existing Storybook content, by migrating (most of) it to VuePress pages (minus the JSX stuff).
